server/tools/peertube-upload.ts

```diff
@@ -105,7 +106,7 @@ async function run () {
 
   console.log('Uploading %s video...', program[ 'videoName' ])
 
-  const videoAttributes = {
+  let videoAttributes:any = {
```

Comment:
You don't need to remove the `const` nor the type inference. You could assign `program['channelId']` beforehand.

Reply:
But what if the user did not supply the channelId on the command line? What default value should we give?

Reply:
It should have value `undefined`.

Reply:
Oh I see, will fix thanks!